src/Mono.Android/Test/Xamarin.Android.RuntimeTests/MainActivity.cs

```diff
@@ -1,16 +1,25 @@
 using System.Reflection;
 using Android.App;
 using Android.OS;
+using Android.Views;
+using Android.Widget;
 using Xamarin.Android.NUnitLite;
 
 namespace Xamarin.Android.RuntimeTests
 {
 	[Activity (Label = "runtime", MainLauncher = true,
 			Name="xamarin.android.runtimetests.MainActivity")]
-	public class MainActivity : TestSuiteActivity
+	public partial class MainActivity : TestSuiteActivity
 	{
 		protected override void OnCreate (Bundle bundle)
 		{
+			// The actual view is set/replaced in `TestSuiteActivity.OnCreate()`
+			SetContentView (Resource.Layout.Main);
+
+			first_text_view.Widget.Click += delegate {
```

Comment:
Question: `Main.axml` contains no `<include/>`s -- which aren't supported yet, anyway -- and only one element with `android:id="@ +id/first_text_view"`.

Why isn't this a *leaf* node? Why do we need the extra `.Widget` here?

Reply:
It's a bug. `first_text_view` should be a leaf element, but not because it's a single one with this particular `id` but because it doesn't have any child elements **with** `android:id`